lib/__tests__/init.test.ts

```diff
@@ -135,4 +135,42 @@ describe("init", () => {
     setUserToken.mockRestore();
     supportsCookies.mockRestore();
   });
+
+  describe("callback for userToken", () => {
```

Comment:
We never test `onUserTokenChange` without the `immediate` option.

Reply:
https://github.com/algolia/search-insights.js/pull/209/commits/f8d2c56c58fafa7eb6cd14885404a6b34917e7d5